.gitignore

```diff
@@ -8,6 +8,7 @@ test/dummy/.sass-cache
 .sass-cache/
 /lit-*.gem
 **.DS_Store
+Gemfile.lock
```

Comment:
Why we have this here: https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/